docs/stability-wasi-proposals-support.md

```diff
@@ -0,0 +1,38 @@
+# WASI Proposals Support
+
+Wasmtime has initial support for some WASI proposals to assist in their
+development as well as allowing users to more easily provide feedback by testing
+them. This is somewhat of a work-in-progress because there aren't a lot of WASI
+proposals at this time.
+
+Currently supported WASI proposals are:
+
+* [The base WASI specification](https://github.com/webassembly/wasi) - the
+  `snapshot` directory is supported by the [`wasmtime-wasi`
+  crate](https://docs.rs/wasmtime-wasi/0.21.0/wasmtime_wasi/)
+
+* [The wasi-nn proposal](https://github.com/WebAssembly/wasi-nn) - the
+  `ephemeral` directory with the wasi-nn APIs.
+
+Wasmtime strives to support WASI proposals in-tree to assist with
+maintainability and provide easier to support to users for easier feedback. In
+doing so, however, we have a few guidelines around adding support for a WASI
+proposal to Wasmtime:
+
+* The proposal must be tested and exercised on CI. These tests don't need to be
+  overly exhaustive but a "smoke test" should pass. Additionally CI should not
+  be overly long compared to existing tests run, and it should also strive to be
+  relataively reliable. Wasmtime contributors may disable CI for proposals at
```

Comment:
Nit: "relataively". However, I think we should not soften this too much, and "strive to" already makes it a bit less strong, so perhaps just remove this?